components/BottomSheet/src/MDCBottomSheetTransition.h

```diff
@@ -0,0 +1,50 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <MotionTransitioning/MotionTransitioning.h>
+
+/**
+ Presents the associated view controller as a draggable bottom view.
+
+ The associated view controller can configure the behavior of the transition by setting a
+ preferredContentSize and providing a tracking scroll view.
+
+ The associated view controller will initially be presented anchored to the bottom middle of the
+ device, with its size matching the view controller's preferredContentSize.
+
+ If the trackingScrollView is set, or one was inferred, then the scroll view's content size will
+ determine how much farther the associated view controller can be expanded.
```

Comment:
"how much farther..." are you talking about the maximum height of the view controller?

Reply:
Done.